spec/presenters/description_display_formatter_spec.rb

```diff
@@ -15,7 +15,7 @@
       "Adds \n\n returns" => "<p>Adds </p>\n\n<p> returns</p>",
       "Adds\n\nreturns\nproperly" => "<p>Adds</p>\n\n<p>returns\n<br />properly</p>",
       "Strips <goat>unwanted tags </goat>" => "<p>Strips unwanted tags </p>",
-      "Adds links to http://www.randomurl.org" => "<p>Adds links to <a href=\"http://www.randomurl.org\"><i class=\"fa fa-external-link\" aria-hidden=\"true\"></i>&nbsp;http://www.randomurl.org</a></p>",
+      "Adds links to http://www.randomurl.org" => "<p>Adds links to <a href=\"http://www.randomurl.org\">http://www.randomurl.org</a></p>",
```

Comment:
It's one of those annoying `html_safe` problems. The test passed, but the HTML was being escaped and ultimately displayed on the front end as `<i></i>`. I just decided to fix the more important bug today, figuring the smaller one can wait until next week.

Reply:
Oh thanks for clarification, and thanks for taking care of this!

I am confused how the test passed without `include ActionView::Helpers::UrlHelper`, if `include ActionView::Helpers::UrlHelper` was necessary to get `auto_link` method!  That's what made me wonder how the test passed. Why was `include ActionView::Helpers::UrlHelper` necessary then? But maybe text comments isn't the best way to discuss this. 

i am pretty sure there would have been a simple way to deal with HTML escaping issues -- possibly just adding `.html_safe` on the end of the string generated. We could just revert and try to do that, instead of doing more complicated refactor possibly involving the other gem? But we could also use the other gem, sure. (It still might need the `.html_safe` fix!)